addons/udes_stock/views/stock_picking.xml

```diff
@@ -59,7 +59,12 @@
             <!-- origin (Source Document  field changed to required for goods-in -->
             <xpath expr="//field[@name='origin']" position="attributes">
                 <attribute name="attrs">{'required': [('picking_type_code', '=', 'incoming')]}</attribute>
-            </xpath>              
+            </xpath>
+
+            <!-- Add date done to form view -->
+            <xpath expr="//group/group[2]" position="inside">
```

Comment:
Can we add it after the scheduled_date.
`<xpath expr="//field[@name='scheduled_date']" position="after">`